src/indev/lv_indev.h

```diff
@@ -42,6 +42,7 @@ typedef enum {
 } lv_indev_state_t;
 
 typedef enum {
+    LV_INDEV_MODE_NONE = -1,
```

Comment:
On other places NONE is usually mapped to 0. Can we do it here as well?

Reply:
Yes please, enums should start always from 0, and in this case we can simply shift other items.